packages/main/src/MultiComboBox.ts

```diff
@@ -1061,11 +1095,11 @@ class MultiComboBox extends UI5Element {
 			}
 
 			innerInput.setSelectionRange(matchingItem.text.length, matchingItem.text.length);
-			this.allItemsPopover.close();
+			this.allItemsPopover?.close();
 		}
 	}
 
-	_resetValueState(valueState, callback) {
+	_resetValueState(valueState: ValueState, callback?: () => any) {
```

Comment:
since we are not using the value of callback maybe it's a good idea to make its return type to void